Event/MessageListEvent.php

```diff
@@ -63,6 +63,10 @@ public function getMax()
      */
     public function getMessages()
     {
+        if (null !== $this->max && is_int($this->max)) {
```

Comment:
I thought about that as well. But if we do it (which might be a BC break for some weird use-cases and should be documented), we should do it in the Tasks and Notifications events as well.

Actually I think the $max logic does not belong into the Event at all, as the backend should filter the list already. I just hesitated to remove it for BC reasons.

Reply:
You're absolutely right. I didn't think about the BC break. I just thought that having the $max without using it was...bad :-P

Reply:
Yes, I am totally with you. It was always there when I forked the original repo and I don't know what it was meant to do. Probably exactly what you did, but then someone stopped in the middle of the implementation... ;-)
- We can keep it as you did or revert it.
- But if we keep it (thats what I would prefer) I would ask you to apply the changes to all events, otherwise one event is behaving different than the others.

Reply:
Ok, I'll keep it and I'll apply the changes in notification and in task list. But... how can we handle the BC break ?

Reply:
And maybe, the is_int is not very usefull....

Reply:
I had another look and the constructor says `* @param int $max Maximun number of notifications displayed in panel`. 
So its quite unlikely that someone will run into an issue here.
I am fine with simply adding a BC break warning in the release notes here at GitHub I'll take care of that).

Reply:
Oh no, its get weirder, check these lines:
https://github.com/kevinpapst/AdminLTEBundle/blob/master/Controller/NavbarController.php#L32
https://github.com/kevinpapst/AdminLTEBundle/blob/master/Controller/NavbarController.php#L55
The $max parameter is not even passed to the constructor, so its only a problem for TaskListEvent. 
Nobody ever complained, I double my opinion: mention it in the release notes is enough.
I guess if you need the feature, you need to fix the Controller as well ;-)

Reply:
Just wanted to ask if you saw my last comment about the controller? Just pass in these 2 max calls and the PR can be merged

Reply:
Sorry, I'didn't understand that. I'll do it right away!

Reply:
Uhoh, I just saw that I will run into an issue with that, as I in my own app I expect the menu to display 10 entries ... LOL

Shall we raise the limit in the controller from 5 to something higher like 20 by default?

Reply:
Let me try to see if we can fix it with a configuration parameter in admin_lte.yaml. In this way we can change its behaviour more easly.

Reply:
Thanks for the update. I will have to have another look ... now I wonder if more people limited the amount of displayed entries in their backend and expect the bundle to render all entries. 
Probably we should make sure the controller passes the limit given to the function to the event constructor (you did that already) but setting the default to null, as null means displaying all events. Whats your opinion?

Reply:
I'm passing the parameters only in test. On production...I would have no control over that...I think.
With this last commit, you can configure the maximum number of elements displayed. I was thinking to extend the commit with some tests.
Anyway, now, if you pass null to the constructor, all elements in the list are displyed (as stated in the tests).
I should also update the documentation. But I'd like to have your opinion before continuing.

Reply:
Tests are always great! But that package is no reference for tests, so you don't have to... but If you want you get extra Karma points :-D 

> I'm passing the parameters only in test. On production...I would have no control over that...I think.
Not sure what you refer to. You mean the $max parameter to the single controller functions? These could only be changed by adjusting the twig call, e.g. 
```
{% block navbar_messages %}
    {{ render(controller('AdminLTEBundle:Navbar:messages', {'max' : 7})) }}
{% endblock %}
```
I think your approach is much better, so users can change the config and don't need to overwrite the blocks.